### PR TITLE
Account kit nilcheck

### PR DESCRIPTION
**Pull request by trivielldev**

### CODE REVIEW

File: src/ios/AKPlugin.m [hunk 1]
```diff
@@ -60,8 +60,19 @@ - (void)loginWithEmail:(CDVInvokedUrlCommand *)command {
 }
 
 - (void)getAccount:(CDVInvokedUrlCommand *)command {
+  if (_accountKit == nil) {
+      _accountKit = [[AKFAccountKit alloc] initWithResponseType:AKFResponseTypeAccessToken];
```
Comment (gurisko) [inline src/ios/AKPlugin.m]: Shouldn't we return an error in such case? `getAccount` should always be called after any of the login methods imho (which initialize `_accountKit`)

**Review by gurisko — COMMENTED** [review]
Thank you @trivielldev for the PR!

File: src/ios/AKPlugin.m (hunk 1 above)
Comment (trivielldev) [inline src/ios/AKPlugin.m]: We want to check if the user has already logged in once in the app. If you always return an error when _accountKit is not initialised then you will force the user to log in once more with SMS even though the tokens are cached and present.

At least this is my experience when testing with the emulator, stopping and starting the app.

Comment (gurisko) [inline src/ios/AKPlugin.m]: I can't test this at the moment (missing the hardware on the holidays). But the thing I don't like is the fact that we are initializing `_accountKit` with access token response. What if user previously logged in with authorization code? Have you experienced the scenario?

Comment (trivielldev) [inline src/ios/AKPlugin.m]: I only did a quick fix to make it work for our use case. I have not done any thorough to see that this fix doesn't break other use cases. We have solved this temporarily and are using our own custom version for now.
I can see if I get a chance to test some other use cases.

**Comment by gurisko** [comment]
Closing due to inactivity.

**Pull request closed without merging**